Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -158,6 +158,20 @@ IPCCommandResult ES::GetTitleID(const IOCtlVRequest& request)
   return GetDefaultReply(IPC_SUCCESS);
 }
 
+static bool UpdateUIDAndGID(const IOS::ES::TMDReader& tmd)
+{
+  IOS::ES::UIDSys uid_sys{Common::FromWhichRoot::FROM_SESSION_ROOT};
+  const u32 uid = uid_sys.GetOrInsertUIDForTitle(tmd.GetTitleId());
```

Comment:
Should the title ID be stored to a const u64? In the event of the failure case, you'd end up doing an unnecessary swap.